handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,115 @@
+
+### Increasing the index cache size for *single node* deployments
+
+Increase the index cache size to process more data and use the Test Insights or Asset Management modules in a single-node deployment. This avoids high CPU usage that can occur with the default MongoDB memory constraints in SystemLink. NI recommends increasing the available memory for the index to 8GB. The following steps can be used to increase the available memory for the MongoDB index.
+  
+1. With a user with administrator privileges, log into the desktop of the SystemLink app server.
+1. Open **NI SystemLink Server Configuration**.
+1. Navigate to **NoSqlDatabase**.
+1. Under **The NoSqlServer is launched and managed by SystemLink**, adjust the value in **The cache size of the database in Gigabytes (GB)**.
+1. Click **Apply**.
+
+!!!note
+    Refer to [Sizing a SystemLink server](https://www.ni.com/documentation/en/systemlink/latest/setup/sizing-a-systemlink-server/) for guidance on the CPU, memory, and disk needed for SystemLink single node deployments.
+
+## Connecting to MongoDB in a Multi Node Configuration
+
+SystemLink supports three types of multi-node configuration. You can configure SystemLink to connect to a standalone instance, replica sets, or MongoDB Atlas.
+
+Connecting SystemLink to a three or more node MongoDB replica set is the recommended configuration for all production deployments.
+
+!!!note
+    While SystemLink can connect and use a sharded MongoDB cluster (`mongos`) it will not take full advantage of the capabilities enabled by sharded clusters.
+
+- NI does not support configurations where the MongoDB instance is used by multiple SystemLink servers.
+- NI does not support configurations where the MongoDB instance is also used by another production application.
+
+!!!important
+    Due to a bug in the Elixir MongoDB driver, SystemLink cannot connect to MongoDB instances where there is either the `-` or the `_` characters in the MongoDB username provided in the form input or connection string.
+
+When using a stand-alone MongoDB instance, replica set, or Atlas, you have the option to the data stored within the database. This is built into MongoDB Atlas or requires [MongoDB Enterprise Advanced](https://www.mongodb.com/products/mongodb-enterprise-advanced) for stand-alone or replica sets.
+
+### Connecting to Standalone MongoDB instance
+
+Using a separate server to host MongoDB provides for greater reliability and lower resource utilization for your SystemLink server.
+
+You may use the supplied form input in **NI SystemLink Sever Configuration** when connecting to a server hosting a single standalone MongoDB instance. Refer to [Connecting to a Remote Mongo Database](https://www.ni.com/documentation/en/systemlink/latest/setup/remote-mongo-database/) for steps to connect with this method.
+
```

Comment:
For SystemLink to make use of a Mongo instance, it is necessary to create an admin user in a an authSource database (usually admin). It seems like we should document that process somewhere.

Reply:
Good call. Included at the beginning of the section on multi node deployments.